Docs/Differences/PreviousConfiguration.tex

```diff
@@ -2365,6 +2365,17 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
 
   \emph{Note}: This option should be avoided whenever possible. Modern firmware
   usually have compatible AHCI controllers.
+  
+  \item
```

Comment:
This file contains previous version of the configuration, i.e. 0.6.3. It obviously did not have that quirk. Do not edit it.

Reply:
Done